ga4gh/cli.py

```diff
@@ -1496,6 +1498,29 @@ def configtest_main(parser=None):
 ##############################################################################
 
 
+def getNameFromPath(filePath):
+    """
+    Returns the filename of the specified path without its extensions.
+    This is usually how we derive the default name for a given object.
+    """
+    if len(filePath) == 0:
+        raise ValueError("Cannot have empty path for name")
+    fileName = os.path.split(filePath)[1]
+    # We need to handle things like .fa.gz, so we can't use
+    # os.path.splitext
+    ret = fileName.split(".")[0]
+    assert ret != ""
```

Comment:
Can we add an optional argument to `getNameFromPath` that would allow you to provide the possible extensions to expect, and remove only those if specified? So for example:
```
getNameFromPath('GL000248.1.fa.gz.fai', ['fa.gz','fa.gz.fai','fa.gz.gzi', 'json'])
```
would correctly return `GL000248.1` as the filename. (this is a real filename from the GRCh37 reference set).